uwblueprint/sistering
# Add submit signups volunteer avail client call

## Ticket link

Closes #230 



## Implementation description
* Call new query for getting shifts + signups for shift
* Update perms for relevant queries for volunteer page
* Added state for add signup + delete signups for update avail page


## Steps to test
1. Go to route `/volunteer/posting/1/availabilities`
2. Existing signups with notes should be displayed
3. Note states should track
4. Adding signups work and persists on refresh
5. Deleting existing signups work and persists on refresh



## What should reviewers focus on?
* Correctness
* Business logic cleanup (would like suggestions on how to clean up : ))

## Checklist
